README.md

```diff
@@ -1,15 +1,228 @@
+
+|Tool    | Index file(s)   |
+|-------------------|:--------------------------------- |
+|samtools            | .fai                               |
+|bwa            | .amb, .ann, .bwt, .pac, .sa                               |
+|picard            | .dict                              |
+
+When you run the pipeline, you will use the mandatory `--ref` parameter to specify the location and name of the reference.fasta file: 
+```
+--ref /path/to/reference.fasta
+```
+***Note***: You must specify the full path for the reference fasta, even if it is in your working directory.
+
+- You will also need to download 
+
+### 3. Clone this repository 
+
+Download the code contained in this repository with: 
+
+```
+git clone https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf
+```
+
+This will create a directory with the following structure: 
+```
+Somatic-shortV-nf/
+├── LICENSE
+├── README.md
+├── config/
+├── images/
+├── scripts/
+├── main.nf
+├── modules/
+└── nextflow.config
+```
+The important features are: 
+
+* **main.nf** contains the main nextflow script that calls all the processes in the workflow.
+* **nextflow.config** contains default parameters to use in the pipeline.
+* **modules** contains individual process files for each step in the workflow. 
+* **config** contains infrastructure-specific config files (this is currently under development)
+
+### 4. Run the pipeline 
+
+The most basic run command for this pipeline is: 
+
+```
+nextflow run main.nf --input samples.csv --ref reference.fasta  --intervalList_path path_to_intervals --ponvcf pon.vcf.gz
+        
+```
+
+By default, this will generate `work` directory, `results` output directory and a `runInfo` run metrics directory inside the results directory. 
+
+To specify additional optional tool-specific parameters, see what flags are supported by running:
+
+```
+nextflow run main.nf --help 
+```
+
+If for any reason your workflow fails, you are able to resume the workflow from the last successful process with `-resume`. 
+
+
+### 5. Results 
+Once the pipeline is complete, you will find all outputs for each sample in the results directory. **Note** I still need to delete arrange the output files in proper sub-directory structure.
+
+## Infrastructure usage and recommendations
+
+This pipeline has been successfully implemented on [NCI Gadi](https://nci.org.au/our-systems/hpc-systems), [Pawsey Setonix HPC](https://pawsey.org.au/systems/setonix/) and [Pawsey Nimbus cloud](https://pawsey.org.au/systems/nimbus-cloud-service/) using infrastructure-specific configs. These configs can be used to interact with the job scheduler and assign a project code to all task job submissions for billing purposes. You can use the following flags to handle accounting:
+
+* `--whoami` your NCI or Pawsey user name
+* `--setonix_account` the Setonix project account you would like to bill service units to
+* `--gadi-account` the Gadi project account you would like to bill service units to
```

Comment:
Why is there an inconsistent use of underscores and tabs in parameters? Please make consistent.